project/Dependencies.scala

```diff
@@ -19,4 +19,10 @@ object Dependencies {
     "org.scalatest" %% "scalatest" % "3.2.9" % Test cross CrossVersion.binary,
     "org.scalacheck" %% "scalacheck" % "1.15.4" % Test cross CrossVersion.binary
   )
+
+  lazy val scalaCollectionCompat: ModuleID =
```

Comment:
Ad commented above, I'd like to keep  `build.sbt` as static as possibile so here we can have a `val` to contains all dependencies for a specific scala version while in `common` we have all common dependencies. 

```scala
val dependenciesForScala2_12: Seq[ModuleID] = Seq(
  "org.scala-lang.modules" %% "scala-collection-compat" % "2.5.0"
)
```